blueprints/addon/files/.travis.yml

```diff
@@ -37,4 +37,4 @@ install:
 script:
   # Usually, it's ok to finish the test scenario without reverting
   #  to the addon's original dependency state, skipping "cleanup".
-  - ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
+  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
```

Comment:
This will certainly work, I proposed using `npm run` and created a `test-ci` script in `package.json`. It is more in line IMO with best practices, but this is a smaller change. I don't feel super strongly about it.